lang/zh-TW.js

```diff
@@ -0,0 +1,227 @@
+    "download": "下載 Hex 檔案"
+  },
+  "load": {
+    "show-files": "顯示檔案",
+    "load-title": "讀取",
+    "instructions": "拖曳一個 .hex 或 .py 檔案到這裡以開啟它。",
+    "submit": "讀取",
+    "save-title": "儲存",
+    "save-hex": "下載專案 Hex 檔案",
+    "save-py": "下載 Python 腳本",
+    "fs-title": "檔案",
+    "toggle-file": "或瀏覽裝置中的檔案。",
+    "fs-add-file": "新增檔案",
+    "hide-files": "隱藏檔案",
+    "td-filename": "檔案名稱",
+    "td-size": "大小",
+    "fs-space-free": "可用空間",
+    "remove-but": "移除",
+    "save-but": "儲存",
+    "files-title": "專案檔案",
+    "help-button": "檔案説明",
+    "file-help-text": "專案檔案區域顯示程式中引用（include）的檔案，並允許你添加或移除外部 python 模組和其它檔案。要進一步瞭解請參閱",
+    "help-link": "Python 編輯器幫助文件",
+    "invalid-file-title": "無效的檔案類型",
+    "mpy-warning": "這個版本的 Python 編輯器還不支援添加 .mpy 檔案。",
+    "extension-warning": "Python 編輯器只能載入副檔名為 .hex 或 .py 的檔案。"
+  },
+  "languages": {
+    "en": {
+      "title": "英文"
+    },
+    "es": {
+      "title": "西班牙语"
+    },
+    "pl": {
+      "title": "波蘭語"
+    },
+    "hr": {
+      "title": "克羅地亞語"
+    },
+    "zh-CN": {
+      "title": "中文（簡體）"
+    },
+    "zh-HK": {
+      "title": "Chinese (traditional, Hong Kong)"
+    },
+    "zh-TW": {
+      "title": "Chinese (traditional, Taiwan)"
+    }
+  },
+  "static-strings": {
+    "buttons": {
+      "command-download": {
+        "title": "下載一個 hex 檔案以燒錄到 micro:bit 上",
+        "label": "下載"
+      },
+      "command-disconnect": {
+        "title": "斷開 micro:bit 連線",
+        "label": "中斷連線"
+      },
+      "command-flash": {
+        "title": "直接將專案燒錄到 micro:bit",
+        "label": "燒錄"
+      },
+      "command-files": {
+        "title": "讀取/儲存檔案",
+        "label": "讀取/儲存"
+      },
+      "command-serial": {
+        "title": "通過序列埠連接 micro:bit",
+        "label": "開啟序列埠",
+        "title-close": "關閉序列埠連線並返回編輯器",
+        "label-close": "關閉序列埠"
+      },
+      "command-connect": {
+        "title": "連線到 micro:bit",
+        "label": "連線"
+      },
+      "command-connecting": {
+        "title": "正在連線到 micro:bit",
+        "label": "正在連線"
+      },
+      "command-options": {
+        "title": "變更編輯器設定",
+        "label": "Beta 選項"
+      },
+      "command-blockly": {
+        "title": "點擊以 blockly 建立程式碼",
+        "label": "Blockly"
+      },
+      "command-snippet": {
+        "title": "點擊以選擇一個片語（快捷代碼）",
+        "label": "程式碼片段"
+      },
+      "command-help": {
+        "title": "探索有用的資源",
+        "label": "幫助"
+      },
+      "command-language": {
+        "title": "選取語言",
+        "label": "語言"
+      },
+      "command-share": {
+        "title": "建立一個連結共享你的腳本",
+        "label": "分享"
+      },
+      "command-zoom-in": {
+        "title": "放大"
+      },
+      "command-zoom-out": {
+        "title": "縮小"
+      }
+    },
+    "script-name": {
+      "label": "腳本名稱"
+    },
+    "options-dropdown": {
+      "autocomplete": "自動完成",
+      "on-enter": "按 [Enter] 後",
+      "partial-flashing": "快速燒錄",
+      "lang-select": "選擇語言：",
```

Comment:
This string contains an space after the colon, which should probably be removed.

Interestingly it looks like the space is rendered at the other side to the colon?
![image](https://user-images.githubusercontent.com/29712657/72352096-9000dc00-36d9-11ea-935e-0e8aff6679ec.png)

Reply:
Fixed in https://github.com/bbcmicrobit/PythonEditor/pull/316/commits/556bf33cd1e0fdabe373c0e2597a208c2333fb3b

<img width="198" alt="Screenshot 2020-01-15 at 09 13 21" src="https://user-images.githubusercontent.com/31242877/72421339-34d1f680-3778-11ea-9c40-eee97fbea2f0.png">